cmd/goal/asset.go

```diff
@@ -661,6 +668,79 @@ func assetDecimalsFmt(amount uint64, decimals uint32) string {
 	return fmt.Sprintf("%d.%0*d", amount/pow, decimals, amount%pow)
 }
 
+var optinAssetCmd = &cobra.Command{
+	Use:   "optin",
+	Short: "Optin to assets",
+	Long:  "Opt an acocunt in to asset use. An account will begin accepting an asset by issuing a zero-amount asset transfer to itself.",
```

Comment:
nit: maybe change `Opt an acocunt in to asset use` to `Opt in to receive new assets`?

Reply:
Updated to `Opt in to receive a new asset.`